pandas/core/algorithms.py

```diff
@@ -2149,3 +2149,23 @@ def _sort_tuples(values: np.ndarray[tuple]):
     arrays, _ = to_arrays(values, None)
     indexer = lexsort_indexer(arrays, orders=True)
     return values[indexer]
+
+
+def make_duplicates_of_left_unique_in_right(
+    left: np.ndarray, right: np.ndarray
+) -> np.ndarray:
+    """
+    Drops all duplicates values from left in right, so that they are
+    unique in right.
```

Comment:
The code itself looks good, but this sentence isn't clear to a reader without context

Reply:
Improved it?